### DISCUSSION TITLE
Golang

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#83] [General]
# Golang

Check if Golang is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/e3d84143-6962-4a6f-9f00-5f05f79ebe43

**Comment by systemnerd** [comment]
I dont see native ARM support for windows on the downloads page. Please rectify this.

**Reply by OpenSource03** [reply]
![All releases - The Go Programming Language](https://github.com/user-attachments/assets/e968c85d-bc09-40e7-aa9b-c023f7ca5ad5)
Under downloads - other ports (image may not be the best but you can see if you zoom in)
